linera-views/src/context.rs

```diff
@@ -167,11 +167,11 @@ pub trait Context: Clone {
 #[derive(Debug, Default, Clone)]
 pub struct ViewContext<E, S> {
     /// The DB client that is shared between views.
-    store: S,
+    pub store: S,
```

Comment:
Why is this needed? I just made a PR to avoid this

Reply:
Ok, I reverted the change and changed the way the `ViewContext` is build.